lib/src/base-styles/_nx-tables.scss

```diff
@@ -212,6 +212,14 @@
       @include nx-small-icon-btn;
     }
   }
+
+  &:first-child {
+    padding-left: $nx-spacing-md;
+  }
+
+  &:last-child {
+    padding-right: $nx-spacing-md;
```

Comment:
Both of these are being overridden by more specific selectors which also happen to be right

Reply:
> Both of these are being overridden by more specific selectors which also happen to be right

Ah so they are: https://github.com/sonatype/sonatype-react-shared-components/pull/391/commits/61930a1161ab4e54949f41c387fe6cf897802c59